packages/block-editor/src/components/ungroup-button/index.native.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Toolbar,
+	ToolbarButton,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import { Ungroup } from './icon';
```

Comment:
Could we name it `UngroupIcon` or UngroupSVG`?

Reply:
Sure!

Reply:
done